catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>Why did you leave error message in notebook? Moreover, frozendict was never used in this notebook.</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/catboost/catboost/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed